nipype/pipeline/plugins/tests/test_callback.py

```diff
@@ -60,3 +61,51 @@ def test_callback_exception(tmpdir, plugin, stop_on_first_crash):
 
     sleep(0.5)  # Wait for callback to be called (python 2.7)
     assert so.statuses == [("f_node", "start"), ("f_node", "exception")]
+
+
+@pytest.mark.parametrize("plugin", ["Linear", "MultiProc", "LegacyMultiProc"])
+def test_callback_gantt(tmp_path: Path, plugin: str) -> None:
+    import logging
+
+    from os import path
+
+    from nipype.utils.profiler import log_nodes_cb
+    from nipype.utils.draw_gantt_chart import generate_gantt_chart
+
+    log_filename = tmp_path / "callback.log"
+    logger = logging.getLogger("callback")
+    logger.setLevel(logging.DEBUG)
+    handler = logging.FileHandler(log_filename)
+    logger.addHandler(handler)
+
+    # create workflow
+    wf = pe.Workflow(name="test", base_dir=str(tmp_path))
+    f_node = pe.Node(
+        niu.Function(function=func, input_names=[], output_names=[]), name="f_node"
+    )
+    wf.add_nodes([f_node])
+    wf.config["execution"] = {"crashdump_dir": wf.base_dir, "poll_sleep_duration": 2}
+
+    plugin_args = {"status_callback": log_nodes_cb}
+    if plugin != "Linear":
+        plugin_args["n_procs"] = 8
+    wf.run(plugin=plugin, plugin_args=plugin_args)
+
+    with open(log_filename, "r") as _f:
```

Comment:
Out of curiosity, why the `_` prefix?

```suggestion
    with open(log_filename, "r") as _f:
```

Reply:
Habit ― I like to indicate temporary scope things like open files are internal variables, but it's definitely not required to be that way

Reply:
Got it.